src/src/WebServer/JSON.cpp

```diff
@@ -448,6 +448,10 @@ void handle_json()
         stream_next_json_object_value(F("Type"),             getPluginNameFromDeviceIndex(DeviceIndex));
         stream_next_json_object_value(F("TaskName"),         getTaskDeviceName(TaskIndex));
         stream_next_json_object_value(F("TaskDeviceNumber"), Settings.TaskDeviceNumber[TaskIndex]);
+        if (getPluginNameFromDeviceIndex(DeviceIndex) == "Switch input - Switch") {
```

Comment:
Few points of critique:
Constant strings, should be wrapped in the `F()` macro, to make sure they are not stored in memory, but read from flash when used.
`F("Switch input - Switch")`

Second, better try to match the plugin ID nr and not the string.
What if someone updates the name to `F("Switch Input - Switch")`, then this fails without clear notice.

The plugin ID (`1`) does not change.

Reply:
> Second, better try to match the plugin ID nr and not the string.


That’s what i wanted to do in the first place but i couldn’t find it...

Reply:
i thought it would be in the SettingStruct too

Reply:
See my edit: 
So better use `getPluginID_from_TaskIndex(TaskIndex) == 1`

Reply:
no,wait. don't answer that... i think i'll find it myself

Reply:
ohh man... it was the whole time right in front of me  🤦‍♂️